test/e2e-go/features/catchup/catchpointCatchup_test.go

```diff
@@ -82,7 +82,7 @@ func TestBasicCatchpointCatchup(t *testing.T) {
 	if testing.Short() {
 		t.Skip()
 	}
-	a := require.New(t)
+	a := require.New(fixtures.SynchronizedTest(t))
```

Comment:
Does this need to be updated as well?
```
func (ec *nodeExitErrorCollector) Print() {
...
		require.NoError(ec.t, err, ec.messages[i])

```

Reply:
Yes, but the change won't be on that line.